src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -160,7 +162,7 @@ function EVMExecutor({ evmcrispr }) {
 
   const handleOnChangeParameters = useCallback((index, event) => {
     const newValue = event.target.value
-    setParameters(prevState => {
+    setParameters((prevState) => {
       const newArray = [...prevState]
       newArray[index] = newValue
       return newArray
```

Comment:
```suggestion
      return  [...prevState, [index]: newValue]
```

Reply:
hehe @kamikazebr look up.

Reply:
Yah i just put it more clear where and how modify your suggestion, was it you suggested right?

Reply:
I didn't know that we could suggest the change on the code, nice 😎

Reply:
Yah, if you want make try, i can cancel mine. Why if he accept it put my name like co-author, should be your name there.

Reply:
Naa mate, go for it, we are a team, it doesn't matter you does it ;)